### ISSUE TITLE
Warning passing square true

### ISSUE THREAD

**Issue by root-io** [boringdesigners/boring-avatars#25]
```
Warning: Received `false` for a non-boolean attribute `rx`.

If you want to write it to the DOM, pass a string instead: rx="false" or rx={value.toString()}.

If you used to conditionally omit it with rx={condition && value}, pass rx={condition ? value : undefined} instead.
rect
mask
svg
w@webpack-internal:///./node_modules/boring-avatars/build/index.js:1:9579
```
using:
```
    <Avatar
      square={true}
      ...
    />
```

**Comment by josepmartins** [comment]
Hey @root-io, thanks for the heads ups, just fixed it on the `1.5.7` version 👍 🚀

**Comment by root-io** [comment]
Thanks @josepmartins for your fix, I just tried it but the warning is still there because I think `SIZE * 2` returns a number.

Solutions that could work:

 ```rx={`${props.square ? undefined : SIZE * 2}`}```
or
 ```rx={props.square ? undefined : String(SIZE * 2)}```

**Comment by josepmartins** [comment]
Hey @root-io , released the `1.5.8` that should fix the problem (now for real). It seems last time the build wasn't right. Let me know if now works without throwing the warning

**Comment by root-io** [comment]
No more warning with 1.5.8 thanks!
